packages/python/build.sh

```diff
@@ -4,10 +4,11 @@ TERMUX_PKG_DESCRIPTION="Python 3 programming language intended to enable clear p
 TERMUX_PKG_LICENSE="custom"
 TERMUX_PKG_LICENSE_FILE="LICENSE"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=3.11.9
+# We get source file archive from Official Python.org's GitHub Account. Uses branch to keep the things latest.
+TERMUX_PKG_VERSION=3.12  # v3.12.4
```

Comment:
No, omitting micro version will confuse users and not work well with services like repology